resources/edc-checkstyle-config.xml

```diff
@@ -46,6 +46,14 @@
     <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
   </module>
 
+  <module name="RegexpHeader">
+    <property name="severity" value="${enhanced_checks_severity}" default="warning"/>
+    <property name="fileExtensions" value="java"/>
+    <!-- default path is for checkstyle GitHub Action running in docker -->
+    <property name="headerFile" value="${base_dir}/resources/java.header" default="/github/workspace/resources/java.header"/>
```

Comment:
in IDE I see a warning.

`The Checkstyle regexp header file as not found and will be ignored`. is it because of default path ? If yes then can we keep to relative path of repo `resources`  folder.

Reply:
Configure the property in the plug-in (path to root)

Reply:
Relative path won't work:
`CheckstyleException: cannot initialize module RegexpHeader - unable to load header file file:.../.gradle/daemon/7.4.1/resources/java.header`

Reply:
We could also inline the file content in the XML to avoid this. Clunky, but simplifies IDE config.

Reply:
After updating plugin `base_dir` key value to root directory, it work as expected. Maybe worth updating in docs to highlight this.

Reply:
Inlined header file content. Not very readable, but simplifies usage